### ISSUE TITLE
Which nrf52 sdk version?

### ISSUE THREAD

**Issue by manvisdr** [otaviojr/zigbee_light_sensor#2]
Which your sdk version to compile this? , I tried use latest and 3.2 still have error in zboss not found. Thanks

**Comment by otaviojr** [comment]
Hi,

just to make sure, did you follow the README link?

https://www.nordicsemi.com/Software-and-Tools/Software/nRF5-SDK-for-Thread-and-Zigbee

I know that nordic is working on a new SDK that will use Zephyr if I'm not mistaken, I think they are calling it nRF Connect SDK, and, probably this code will not work with it. This code is based on the nRF5-SDK.

Are you using the right one?

regards

**Comment by manvisdr** [comment]
yeah, i dont use zephyr, i use zigbee sdk from nordic page. I use old sdk version which is 3.2 version and still cant to compile and error in zboss. But i try in version 4.0.0 and succeed to compile.
